.travis.yml

```diff
@@ -41,16 +40,21 @@ before_install:
     # - sudo rm -rf /dev/shm
     # - sudo ln -s /run/shm /dev/shm
 
+    # clone larray repository
+    - git clone https://github.com/liam2/larray.git larray
```

Comment:
don't forget to set it to larray-project if you merge this after liam2/larray#600

Reply:
Yes, I know. 
I wanted to run travis first because the few doctests present in larray-eurostat crash on my computer. 
For some reasons, travis didn't start?

Reply:
Travis CI was not configured to build PR for this project. I just changed that.

Reply:
Thanks